proposals/description/tip_121c.ts

```diff
@@ -67,122 +123,156 @@ const tip_121c: TemplatedProposalDescription = {
-        ethers.utils.id('BURN_MINTER_ROLE_TEMPORARY_ROLE') // adminRole
-      ],
-      description: 'Set BURN_MINTER_ROLE_TEMPORARY_ROLE as the admin of MINTER_ROLE.'
+      method: 'setWithdrawInfo(address,(address,address,uint96))',
+      arguments: (addresses) => [addresses.rariPool8FeiPCVDeposit, [addresses.simpleFeiDaiPSM, addresses.fei, '0']],
+      description: 'Update Fuse Withdrawal Guard for FEI destination to simpleFeiDaiPSM'
     },
     {
-      target: 'core',
+      target: 'fuseWithdrawalGuard',
       values: '0',
-      method: 'renounceRole(bytes32,address)',
+      method: 'setWithdrawInfo(address,(address,address,uint96))',
+      arguments: (addresses) => [addresses.rariPool8DaiPCVDeposit, [addresses.simpleFeiDaiPSM, addresses.dai, '0']],
+      description: 'Update Fuse Withdrawal Guard for DAI destination to simpleFeiDaiPSM'
+    },
+
+    /// 5. Send all LUSD, BAL and bbaUSD to Tribal Council Multisig for selling
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
       arguments: (addresses) => [
-        ethers.utils.id('BURN_MINTER_ROLE_TEMPORARY_ROLE'), // role
-        addresses.feiDAOTimelock // adminRole
+        addresses.lusdHoldingPCVDeposit, // from
+        addresses.lusd, // token
+        addresses.tribalCouncilSafe, // to
+        '10000' // basisPoints, 100%
       ],
-      description:
-        "The BURN_MINTER_ROLE_TEMPORARY_ROLE can't be granted anymore, so nobody will ever be able to grant the MINTER_ROLE either."
+      description: 'Withdraw remaining LUSD ~17k to Tribal Council Safe for selling'
     },
 
-    // 3. Final handling of veBAL : tokenize it (for redemptions) and deprecate all related contracts
-    // Grant METAGOVERNANCE_TOKEN_STAKING + PCV_CONTROLLER_ROLE role to an immutable contract (that is an ERC20 of
-    // tokenized veBAL), and have a call in this contract that can be used to unlock veBAL and pull to self
-    // with withdrawERC20, and another to burn the token itself for redeeming B-80BAL-20WETH after they
-    // have been pulled to the contract.
-
-    // 4. Final handling of veANGLE : nothing to do (write it off).
-    // angleDelegatorPCVDeposit has 478,096.47 ANGLE Locked Until Apr 2nd 2026 (3.623 y)
-    // worth $ 22,288 on 18 Aug 2022 + ~5k€ of unclaimed sanUSDC_EUR fees.
-    // angleDelegatorPCVDeposit is not a proxy so we can't change the hardcoded behavior.
-
-    // Finally, Collateralization Oracle and Safe Addresses updates
+    // TODO do these numbers need to be updated and/or can ratio PCV controller be used?
     {
-      target: 'collateralizationOracle',
+      target: 'veBalDelegatorPCVDeposit',
       values: '0',
-      method: 'swapDeposit(address,address)',
-      arguments: (addresses) => [addresses.daiFixedPricePSM, addresses.simpleFeiDaiPSM],
-      description: 'Swap old for new DAI PSM in CR Oracle'
+      method: 'withdrawERC20(address,address,uint256)',
```

Comment:
TODO: Make sure daiHoldingDeposit can't be unpaused. validate() after that it is paused

Reply:
TODO: Make sure fuse withdrawal guard doesn't have daiHoldingDeposit as a target, as it will unpause the daiHoldingDeposit in using the ratioPCVController

Reply:
Ensure it will call `pause()` afterwards